src/App.js

```diff
@@ -1,43 +1,73 @@
 import "./App.css";
-import logo from "./logo.png";
+import { Navbar, RequiresAuth } from "./components/index";
+import { Routes, Route } from "react-router-dom";
+import {
+  Home,
+  History,
+  LikedVideos,
+  Playlists,
+  WatchLater,
+  Explore,
+  LoginPage,
+  SignupPage,
+  VideoDetail,
+  Playlist,
+} from "./pages/index";
```

Comment:
no need to write `index` here , it will automatically refer index.

Reply:
ok will do